tests/fast-integration/audit-log/helpers.js

```diff
@@ -0,0 +1,85 @@
+const { 
+    waitForFunction,
+    k8sApply,
+    kubectlApplyDir,
+    kubectlDeleteDir,
+    sleep
+} = require("../utils");
+const { assert } = require("chai");
+
+const testNS = "audit-test"
+const fnName = "audit-test-fn"
+
+async function createNamespace() {
+    await k8sApply([namespaceObj(testNS)])
+}
+
+
+async function deployK8sResources() {
+    await kubectlApplyDir("./audit-log/fixtures", testNS)
+    await waitForFunction(fnName, testNS, 120000);
+
+}
+
+async function deleteK8sResources() {
+    await kubectlDeleteDir("./audit-log/fixtures", "audit-test")
+}
+
+async function waitForAuditLogs() {
+    await sleep(60000);
+}
+
+function parseAuditLogs(logs, resName, groupName, action) {
+    let found = new Boolean(false)
+    logs.forEach(element => {
+        if (element.message.includes(groupName)) {
+            if (element.message.includes(resName)){
+                if (element.message.includes(action)) {
+                    found = true
```

Comment:
I think this can be refactored, when we have first element that matches our criteria we can break the loop, can't we?